src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -113,17 +128,15 @@ public String getConfirmation(@PathVariable String companyNumber,
 
     private String setUpPaymentDateDisplay(PayableLateFilingPenalty payableLateFilingPenalty) {
         if (payableLateFilingPenalty.getPayment() != null) {
-            return StringUtils.isEmpty(payableLateFilingPenalty.getPayment().getPaidAt()) ? "" :
-                    OffsetDateTime.parse(payableLateFilingPenalty.getPayment().getPaidAt(), DateTimeFormatter.ISO_OFFSET_DATE_TIME)
-                            .format(DateTimeFormatter.ofPattern("d MMMM uuuu", Locale.UK));
+            return LocalDate.now()
+                    .format(DateTimeFormatter.ofPattern("d MMMM uuuu", Locale.UK));
         }
         return "";
     }
 
-    private String setUpPaymentAmountDisplay(PayableLateFilingPenalty payableLateFilingPenalty) {
-        if (payableLateFilingPenalty.getPayment() != null) {
-            return StringUtils.isEmpty(payableLateFilingPenalty.getPayment().getAmount()) ? "" :
-                    "£" + payableLateFilingPenalty.getPayment().getAmount() + " (no VAT is charged)";
+    private String setUpPaymentAmountDisplay(LateFilingPenalty payableLateFilingPenalty) {
+        if (payableLateFilingPenalty.getOriginalAmount() != null) {
+            return "£" + penaltyUtils.getFormattedOutstanding(payableLateFilingPenalty.getOriginalAmount()) + " (no VAT is charged)";
```

Comment:
Would we add in the "£" to the html template instead? See viewPenalties.html for example. To keep it consistent, something like;

                    <tr class="govuk-table__row">
                        <td class="govuk-table__header">Amount</td>
                        <td class="govuk-table__cell" th:text="'£' + ${outstanding} + ' (no VAT is charged)'"></td>
                    </tr>

Reply:
I'll get this into the html instead